src/packagedcode/rubygems.py

```diff
@@ -647,93 +626,73 @@ def normalize(gem_data, known_fields=known_fields):
     )
 
 
-def parse_spec(location):
-    pass
+def build_packages_from_gemspec(location):
+    """
+    Return RubyGem Package from gemspec file.
+    """
+    gemspec_data = spec.parse_spec(location)
+    
+    name = gemspec_data.get('name')
+    version = gemspec_data.get('version')
+    homepage_url = gemspec_data.get('homepage_url')
+    summary = gemspec_data.get('summary')
+    description = gemspec_data.get('description')
+    if len(summary) > len(description):
+        description = summary
+
+    declared_license = gemspec_data.get('license')
+    if declared_license:
+        declared_license = declared_license.split(',')
```

Comment:
Just curious: is this the podspec spec to have license always separated by comma?

Reply:
No, not always separated by the comma. I have used this to make it a list because I was getting the correct license_expression value after doing this. I will look at this again :)